src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/setup-prometheus-agent.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Set up the Prometheus agent
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure Prometheus Configurator
+metaDescription: Set up the Prometheus agent
+---
+
+You need to place the examples below in the `config` section of the agent. Refer to the [installation method](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent) you used to known where that is in your case.
+
+## Prevent billing increases [#rate-limits]
+
+Avoid sending Prometheus data that isn't relevant to your monitoring needs. Instead, use filters to ignore or include specific metrics, which help you control the amount of data you send to New Relic and also avoid additional billing charges.
+
+As explained in the [installation guide](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent), by default we set up the Prometheus Kubernetes discovery and scrape everything annotated with `prometheus.io/scrape: true`, and send all the data that's exposed from those discovered targets. In consequence, you may exceed New Relic's platform limits and increase your billing charges.
+
+Also, see our [troubleshooting tips](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/troubleshooting-guide).
+
+## Prometheus Integrations [#prometheus-integrations]
```

Comment:
I guess this section is simply a placeholder, adding a comment not to forget about it